pallets/gear/src/tests.rs

```diff
@@ -76,6 +76,20 @@ fn unstoppable_block_execution_works() {
             EMPTY_PAYLOAD.to_vec(),
             0,
             true,
+            false,
+        )
+        .expect("calculate_gas_info failed");
+
+        let GasInfo {
+            burned: subsequent_burned_gas,
+            ..
+        } = Gear::calculate_gas_info(
+            USER_1.into_origin(),
+            HandleKind::Handle(program_id),
+            EMPTY_PAYLOAD.to_vec(),
+            0,
+            true,
+            true,
```

Comment:
I see no reason to add this field to rpc call. We have no guarantees when message will be executed, so to calculate it properly, need to add this value in min_limit by default and provide some filed like `might_be_returned` with amount of gas which might not be burned due to some conditions